<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>567</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>-30</x>
     <y>0</y>
     <width>811</width>
     <height>561</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Standariser Time Serie</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PAA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SAX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>220</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MoolBoran</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre d'intervalles</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>330</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MoolBoran</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Nombre de symboles</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vis_paa">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>88</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 150, 150);
border-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius : 5px;
border-width : 0px;
padding: 4px;
background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -1.0, radius: 1.35, stop: 0 #fff, stop: 1 #2e3333);
min-width: 80px; </string>
    </property>
    <property name="text">
     <string>Visualiser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vis_sax">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>360</y>
      <width>88</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 150, 150);
border-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius : 5px;
border-width : 0px;
padding: 4px;
background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -1.0, radius: 1.35, stop: 0 #fff, stop: 1 #2e3333);
min-width: 80px; </string>
    </property>
    <property name="text">
     <string>Visualiser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="vis_stand">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>140</y>
      <width>88</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 150, 150);
border-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius : 5px;
border-width : 0px;
padding: 4px;
background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -1.0, radius: 1.35, stop: 0 #fff, stop: 1 #2e3333);
min-width: 80px; </string>
    </property>
    <property name="text">
     <string>Visualiser</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Reference Sans Serif</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Apriori Algorithm</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>430</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MoolBoran</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Min_Support</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resultat">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>460</y>
      <width>88</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 150, 150);
border-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius : 5px;
border-width : 0px;
padding: 4px;
background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -1.0, radius: 1.35, stop: 0 #fff, stop: 1 #2e3333);
min-width: 80px; </string>
    </property>
    <property name="text">
     <string>Result</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>190</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>300</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>400</y>
      <width>711</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="browseline">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>411</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="nbpaa">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>250</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nbsax">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nbstandar">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MoolBoran</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Numéro de série</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MoolBoran</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Numéro de série</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>201</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>MoolBoran</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Numéro de série</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="paaline">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>250</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="saxline">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>360</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="aprioriline">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Standaris">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>88</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 150, 150);
border-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius : 5px;
border-width : 0px;
padding: 4px;
background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -1.0, radius: 1.35, stop: 0 #fff, stop: 1 #2e3333);
min-width: 80px; </string>
    </property>
    <property name="text">
     <string>Standaris</string>
    </property>
   </widget>
   <widget class="QPushButton" name="paa">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>250</y>
      <width>88</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 150, 150);
border-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius : 5px;
border-width : 0px;
padding: 4px;
background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -1.0, radius: 1.35, stop: 0 #fff, stop: 1 #2e3333);
min-width: 80px; </string>
    </property>
    <property name="text">
     <string>PAA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sax">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>88</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 150, 150);
border-color: rgb(0, 0, 0);
border-color: rgb(255, 255, 255);
color: rgb(255, 255, 255);
border-radius : 5px;
border-width : 0px;
padding: 4px;
background: qradialgradient(cx: 0.3, cy: -0.4, fx: 0.3, fy: -1.0, radius: 1.35, stop: 0 #fff, stop: 1 #2e3333);
min-width: 80px; </string>
    </property>
    <property name="text">
     <string>SAX</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>4</width>
     <height>12</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
